jira_integration_example/utilities/pubsub.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2020 Google, LLC.
```

Comment:
Discussed offline: When running the main application in one of the jira / philips hue example folders, the root directory of the repository is not included in the sys.path resulting in packages in the root not being importable. We decided to look into symlinks to solve the problem and that if it wouldn't work, it would be acceptable to just duplicate the pubsub.py and secrets.py files. I looked into symlinks and found that it wouldn't work since Docker doesn't support symlinking to files outside the build context, so I am going to continue with duplicating the files for each integration example for the time being.